buildtools/commands/update-readme.js

```diff
@@ -0,0 +1,28 @@
+const { resolve, join } = require('path');
+const shell = require('shelljs');
+const replace = require('replace-in-file');
+
+const ROOT_DIR = resolve(__dirname, '../../');
+
+function getPublishedWidgetVersion () {
+  const stdout = shell.exec('yarn info @okta/okta-signin-widget versions', { silent: true });
+  const arrayStr = stdout.substring(stdout.indexOf('['), stdout.lastIndexOf(']') + 1).replace(/'/g, '"');
```

Comment:
Why not use the --json option to `yarn info`?

Reply:
```
yarn info @okta/okta-signin-widget versions --json
{"type":"warning","data":"../package.json: No license field"}
{"type":"inspect","data":["1.4.0","1.5.0","1.6.0-rc.0","1.6.0","1.7.0","1.8.0","1.9.0","1.11.0","1.12.0","1.13.0","2.1.0","2.3.0","2.4.0","2.5.0","2.6.0","2.7.0","2.8.0","2.9.0","2.10.0","2.11.0","2.12.0","2.13.0","2.14.0","2.15.0","2.16.0","2.17.0","2.18.0","2.19.0","2.20.0","2.20.1","2.21.0","3.0.0","3.0.1","3.0.2","3.0.3","3.0.4","3.0.5","3.0.6","3.1.0","3.1.1","3.1.2","3.1.3","3.1.4","3.1.5","3.1.6","3.2.0","3.2.1","3.2.2","3.3.0","3.3.1","3.3.2","3.3.3","3.4.0","3.4.1","3.4.2","3.5.0","3.6.0","3.6.1","3.6.2","3.7.0","3.7.1","3.7.2","3.7.3","3.8.0","3.8.1","3.8.2","3.9.0","3.9.1","3.9.2","4.0.0","4.0.1","4.0.2","4.1.0","4.1.1","4.1.2","4.1.3","4.1.4","4.1.5","4.2.0","4.2.1","4.2.2","4.2.3","4.2.4","4.2.5","4.2.6","4.2.7","4.3.0","4.3.1","4.3.2","4.3.3","4.3.4","4.3.5","4.4.0","4.4.1","4.4.2","4.4.3","4.4.4","4.5.0","4.5.1","4.5.2","5.0.0","5.0.1","5.0.2","5.1.0","5.1.1","5.1.2","5.1.3","5.1.4","5.1.5","5.2.0","5.2.1","5.2.2","5.2.3","5.2.4","5.3.0","5.3.1","5.3.2","5.3.3"]}
```
Due to the "no license field" two JSON objects are actually returned. So it's not any easier to work with. For comparison the non --json output looks like:

```
[
  '1.4.0',
  '1.5.0',
  '1.6.0-rc.0',
  '1.6.0',
  '1.7.0',
  '1.8.0',
  '1.9.0',
  '1.11.0',
  '1.12.0',
  '1.13.0',
  '2.1.0',
  '2.3.0',
  '2.4.0',
  '2.5.0',
  '2.6.0',
  '2.7.0',
  '2.8.0',
  '2.9.0',
  '2.10.0',
  '2.11.0',
  '2.12.0',
  '2.13.0',
  '2.14.0',
  '2.15.0',
  '2.16.0',
  '2.17.0',
  '2.18.0',
  '2.19.0',
  '2.20.0',
  '2.20.1',
  '2.21.0',
  '3.0.0',
  '3.0.1',
  '3.0.2',
  '3.0.3',
  '3.0.4',
  '3.0.5',
  '3.0.6',
  '3.1.0',
  '3.1.1',
  '3.1.2',
  '3.1.3',
  '3.1.4',
  '3.1.5',
  '3.1.6',
  '3.2.0',
  '3.2.1',
  '3.2.2',
  '3.3.0',
  '3.3.1',
  '3.3.2',
  '3.3.3',
  '3.4.0',
  '3.4.1',
  '3.4.2',
  '3.5.0',
  '3.6.0',
  '3.6.1',
  '3.6.2',
  '3.7.0',
  '3.7.1',
  '3.7.2',
  '3.7.3',
  '3.8.0',
  '3.8.1',
  '3.8.2',
  '3.9.0',
  '3.9.1',
  '3.9.2',
  '4.0.0',
  '4.0.1',
  '4.0.2',
  '4.1.0',
  '4.1.1',
  '4.1.2',
  '4.1.3',
  '4.1.4',
  '4.1.5',
  '4.2.0',
  '4.2.1',
  '4.2.2',
  '4.2.3',
  '4.2.4',
  '4.2.5',
  '4.2.6',
  '4.2.7',
  '4.3.0',
  '4.3.1',
  '4.3.2',
  '4.3.3',
  '4.3.4',
  '4.3.5',
  '4.4.0',
  '4.4.1',
  '4.4.2',
  '4.4.3',
  '4.4.4',
  '4.5.0',
  '4.5.1',
  '4.5.2',
  '5.0.0',
  '5.0.1',
  '5.0.2',
  '5.1.0',
  '5.1.1',
  '5.1.2',
  '5.1.3',
  '5.1.4',
  '5.1.5',
  '5.2.0',
  '5.2.1',
  '5.2.2',
  '5.2.3',
  '5.2.4',
  '5.3.0',
  '5.3.1',
  '5.3.2',
  '5.3.3',
  '5.4.0'
]
```

Reply:
That's....interesting.  It doesn't on my machine, good to know, I guess.

Reply:
turns out the warning is going to stderr, so there is only one JSON object. switching to --json